ui/src/main/webapp/package.json

```diff
@@ -60,23 +68,13 @@
     "showdown": "^1.5.1",
     "text-loader": "^0.0.1",
     "webpack-merge": "^0.15.0",
-    "zone.js": "^0.7.6",
-    "d3": "4.4.0",
-    "d3-transition": "1.0.3",
-    "d3-selection-multi": "1.0.0",
-    "ngx-charts": "^3.0.2",
-    "angular2-moment": "^1.1.0"
+    "zone.js": "^0.7.6"
```

Comment:
Could you please keep the previous order, it is hard to see what are the actual changes 
(and there will be some potential merge conflicts, since there are couple of other PRs changing dependencies).

I think we should reorder dependencies alphabeticaly (as it npm and maybe yarn too does when installing new dependencies using it) but we should do that in separate PR and without actually changing dependencies or versions.